portfolio/src/main/webapp/index.html

```diff
@@ -58,13 +71,18 @@ <h4>Mili - Camelia (violin cover)</h4>
             <iframe width="280" height="157" src="https://www.youtube.com/embed/pMvMs_tn4as" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
           </div>
       </div>
-      <br>
+      <br><br><br>
 
-      <form action="/form-handler" method="POST">
-        <h2>Contact me</h2>
-        <p class="contactfield">Name &nbsp; &nbsp; <br><textarea name="name-input"></textarea></p>
-        <p class="contactfield">Email &nbsp; &nbsp; <br><textarea name="email-input"></textarea></p>
-        <p class="contactfield">Message &nbsp; &nbsp; <br><textarea name="message-input"></textarea></p>
+      <form style="width:50%; margin:auto" autocomplete="off" action="/form-handler" method="POST" id="contact">
```

Comment:
Made the contact form look better and required the client to fill in each field.